src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+
+const { getFlipperId } = require('../../feature-toggles/helpers');
+
+describe('feature-toogles', () => {
+  describe('Create a cookie', () => {});
```

Comment:
I'd recommend making these sub `describe`s the function names. Then you're structured like:
```js
describe('feature toggles', () => {
  describe('setCookie', () => {
    it('should return the cookie if it already exists', () => {});
    it('should set a new cookie if one doesn't already exist, () => {});
  });

  describe('createTokenFromCookie', () => {
    it('should return the cookie if it already exists', () => {});
    it('should return a random string if the cookie doesn't already exist', () => {});
  });
});
```